src/assets/images/guppy-logo.svg

```diff
@@ -0,0 +1,146 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
You'll notice there's two copies of this file: the one here, where you'd expect in `serc/assets/images`, but also one in `public/`.

The reason for this is that we need the logo within the HTML, to show while React bootstraps.

Because assets are bundled with Webpack, the `src` files will have a hash suffix:
<img width="411" alt="screen shot 2018-08-22 at 3 25 19 pm" src="https://user-images.githubusercontent.com/6692932/44485752-99d6fa80-a61f-11e8-8e93-894e15d8010e.png">

So, sadly, I think I need to duplicate the asset, which is no fun :/ since yeah I COULD hardcode it based on the current hash, but then if the logo changes it would be very easy to forget, and then have a broken image on the loading screen.

Hm, actually... we could pretty easily disable these hashes entirely for images. I wonder if there's a way for us to entirely disable the browser cache, so that we don't need the hashes...

Reply:
You could also use a symlink from the source to the `public/` directory (or vice versa).

Reply:
> You could also use a symlink from the source to the public/ directory (or vice versa).

Interesting - are symlinks cross-platform-friendly? Not sure how they'd work on Windows...

Reply:
> Not sure how they'd work on Windows...

Ah, they might not work there :/